docs/contributions/0_contribution_guide.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Contribution Guide
+nav_order: 2
+has_children: true
+---
+
+# STAR.VOTE contribution guide
+
+We're so excited you're interested in contributing!
+
+Start by joining our community [here](https://starvoting.us/join). Once you fill out the form you'll receive an invite to our slack where you can coordiate with the community
+
+You can also view our calendar [here](https://www.starvoting.org/calendar) and join our weekly software meetings. 
+ 
+Once you're ready to start contributing you can go to [the next step](contributions/setup_locally.html) for building your local setup
```

Comment:
This link doesn't work, missing the "1_". A few other links aren't working as well.

Reply:
Fixed! (and did a pass fixing a few others as well)